src/scripts/my-manager.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+[ -f /etc/environment ] && source /etc/environment
+ulimit -c ${ULIMIT_CONFIGURATION:0}
+name="my-manager"
+
+case $1 in
+    start|boot)
+        ${name} -D
+        ;;
+    stop|shutdown)
+        if [ -f /var/run/${name}.pid ]; then
+            kill `cat /var/run/${name}.pid`
+        else
+            killall ${name}
+        fi
+        ;;
+    debuginfo)
+        ubus-cli "Obj.?"
+        ;;
+    restart)
+        $0 stop
+        sleep 1s
```

Comment:
sleep is usually bad idea anywhere. Sometimes it is used to workaround some race conditions for some asynchronous operations. In this case, stop is synchronous, so no need to use sleep